lib/application-processing/resolvers.ts

```diff
@@ -331,7 +331,7 @@ export const completeApplication: Resolver<
                     },
                   },
                 }
-              : { guardian: { delete: true } }),
+              : {}),
```

Comment:
Should we still account for the case where there was previously a guardian, but the new application does not contain a guardian? There should still be a delete right?

Reply:
Addressed this case by adding disconnect.

Reply:
From what I understand this removes the mapping between the permit and the guardian, but there will be an orphaned guardian in the DB right? Ideally that should be cleaned up too.